cognite/extractorutils/uploader/_metrics.py

```diff
@@ -24,22 +24,13 @@
     "cognite_raw_uploader_rows_duplicates", "Total number of duplicates found", labelnames=["destination"]
 )
 RAW_UPLOADER_QUEUE_SIZE = Gauge("cognite_raw_uploader_queue_size", "Internal queue size")
-RAW_UPLOADER_LATENCY = Histogram(
```

Comment:
Not clear to me why you removed these, it's something people will be asking for again.

Reply:
IIRC, it's something we added to prove something to one customer, and have been stuck with since. 

It is causing a bunch of headache, especially with typing. Which is why all the other builds are failing.